template/template.go

```diff
@@ -39,13 +39,14 @@ type Template struct {
 
 // Environment variables that could be defined to override default behaviors.
 const (
-	EnvAcceptNoValue    = "GOTEMPLATE_NO_VALUE"
-	EnvStrictErrorCheck = "GOTEMPLATE_STRICT_ERROR"
-	EnvSubstitutes      = "GOTEMPLATE_SUBSTITUTES"
-	EnvDebug            = "GOTEMPLATE_DEBUG"
-	EnvExtensionPath    = "GOTEMPLATE_PATH"
-	EnvInternalLogLevel = "GOTEMPLATE_INTERNAL_LOG_LEVEL"
-	EnvLogLevel         = "GOTEMPLATE_TEMPLATE_LOG_LEVEL"
+	EnvAcceptNoValue        = "GOTEMPLATE_NO_VALUE"
+	EnvStrictErrorCheck     = "GOTEMPLATE_STRICT_ERROR"
+	EnvSubstitutes          = "GOTEMPLATE_SUBSTITUTES"
+	EnvSubstitutesSeparator = "GOTEMPLATE_SUBSTITUTES_SEPARATOR"
```

Comment:
I don't think that this feature is important enough to justify an environment variable. I would go for a fixed one.

Reply:
I removed it